### PR TITLE
Fix port type and argument checking

### PR DESCRIPTION
**Pull request by LuQQiu**

### CODE REVIEW

File: alluxio/alluxio_file_system.py [hunk 1]
```diff
@@ -105,8 +105,8 @@ def __init__(
         options=None,
         logger=None,
         concurrency=64,
-        http_port="28080",
-        etcd_port="2379",
+        etcd_port=None,
+        worker_http_port=None,
```
Comment (ChunxuTang) [inline alluxio/alluxio_file_system.py:109]: Probably need to update the tests and fsspec.

File: alluxio/worker_ring.py [hunk 2]
```diff
@@ -205,19 +205,19 @@ class ConsistentHashProvider:
     def __init__(
         self,
         etcd_hosts=None,
-        etcd_port=2379,
+        etcd_port=None,
         worker_hosts=None,
-        worker_http_port=DEFAULT_HTTP_SERVER_PORT,
+        worker_http_port=None,
```
Comment (jja725) [inline alluxio/worker_ring.py:210]: I feel we can just use list of WorkerNetAddress here, What do you think

Comment (LuQQiu) [reply alluxio/worker_ring.py:210]: WorkerNetAddress has too many legancy fields, most of the fields are not actually being used or are not related to current Alluxio python api

Comment (LuQQiu) [reply alluxio/worker_ring.py:210]: Later refactor PR can consider how to refractor this part of code and separate the responsibilities

**Pull request merged**